RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -222,7 +225,10 @@ public RuntimeService getRuntimeService() {
         LinkedList<ExecutionContext> newRuns = new LinkedList<>();
         for (ExecutionContext oldContext : contexts) {
             DataSet ds = oldContext.getDataSet();
-            if (!test && !checkJobStartability(ds)) continue;
+            if (!test && !canStartJobs(ds)) {
```

Comment:
I see why you did this, but now you have code duplication

Reply:
Could be, but in this situation the method is a simple checker and should not have any side effects. If we want to remove the single line code duplication (the method is called at two places), we could introduce another method of at least two lines. So here I think functional stringency over minor code duplication.